packages/paste-website/src/components/shortcodes/table-of-contents/styles.ts

```diff
@@ -0,0 +1,63 @@
+import Scrollspy from 'react-scrollspy';
+import styled from '@emotion/styled';
+import {themeGet} from 'styled-system';
+import {Box} from '@twilio-paste/box';
+import {TableOfContentsListItemProps} from './types';
+
+export const StyledWrapper = styled(Box)`
+  order: 2;
+`;
+
+export const StyledScrollSpy = styled(Scrollspy)`
+  position: sticky;
+  top: 0;
+  right: 0;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+`;
+
+export const marginTopPartial = ({depth}: TableOfContentsListItemProps): string => {
+  if (depth === '3' || depth === '4') {
+    return '0';
+  }
+  return themeGet('space.space30');
+};
+
+export const marginLeftPartial = ({depth}: TableOfContentsListItemProps): string => {
+  if (depth === '3' || depth === '4') {
+    return themeGet('space.space50');
+  }
+  return '0';
+};
+
+export const fontSizePartial = ({depth}: TableOfContentsListItemProps): string => {
+  if (depth === '3' || depth === '4') {
+    return themeGet('fontSizes.fontSize10');
+  }
+  return themeGet('fontSizes.fontSize20');
+};
+
+export const StyledListItem: React.FC<TableOfContentsListItemProps> = styled.li`
+  display: block;
+  margin-top: ${marginTopPartial};
+  margin-left: ${marginLeftPartial};
+  padding-left: ${themeGet('space.space30')};
+  font-size: ${fontSizePartial};
+  color: ${themeGet('colors.colorGray80')};
+  border-left-width: 2px;
+  border-left-style: solid;
+  border-left-color: transparent;
+
+  a {
```

Comment:
Let's avoid descendants like this. You are trying to override the styles of the Anchor component and not honouring component encapsulation.

I'd actually like to request a design change @two24studios, our links / clickable thing are blue, can we keep them blue so it's consistent? Then we don't even need to override the styles

Reply:
I'm on board with that. I'll chat with @two24studios about looking at the design. Removing the style overrides for now.